internal/jobcontainers/process.go

```diff
@@ -73,7 +74,7 @@ func (p *JobProcess) Signal(ctx context.Context, options interface{}) (bool, err
 	defer p.procLock.Unlock()
 
 	if p.exited() {
-		return false, errors.New("signal not sent. process has already exited")
+		return false, errors.Wrap(hcs.ErrProcessAlreadyStopped, "signal not sent. process has already exited")
```

Comment:
I honestly think we should probably just return `true, nil` here if it's already exited, thoughts?

Reply:
That's fair, I'm remembering somewhere else we followed just returning true, nil on Kill: https://github.com/microsoft/hcsshim/blob/master/internal/hcs/process.go#L157-L163. But that's if we verified we already called TerminateProcess ourselves which is a bit different

Reply:
Okay, I'm convinced 😄